api/organisations/invites/views.py

```diff
@@ -44,6 +46,15 @@ def join_organisation_from_email(request, hash):
         error_data = {"detail": str(e)}
         return Response(data=error_data, status=status.HTTP_400_BAD_REQUEST)
 
+    hubspot_cookie = request.COOKIES.get(HUBSPOT_COOKIE_NAME)
+
+    if hubspot_cookie:
+        if not HubspotTracker.objects.filter(user=request.user).exists():
+            HubspotTracker.objects.create(
+                user=request.user,
+                hubspot_cookie=hubspot_cookie,
+            )
+
```

Comment:
I'd like to avoid code duplication, which occurs further in `join_organisation_from_link`. This code can be extracted to either of the following:
- A `HubspotTracker` class method
- A private method in this module — dirty, but gets the job done
- A function in `services.py` — that's how I'd do it, see existing `services.py` modules for reference

Reply:
Sure this is easy.